### PR TITLE
Add various in-code documentation to ApiServer

### PR DESCRIPTION
**Pull request by StachuDotNet**
## What is the problem/goal being addressed?
Increase in-code documentation of ApiServer, for future developers' understanding.

## What is the solution to this problem?
- adjust existing commentary from //-style to ///-style, so IDEs show the the descriptions embedded in their UIs.
  image
- add additional commentary where relevant

### CODE REVIEW

**Comment by StachuDotNet** [comment]
I started with ApiServer as it's relatively small. I hope to follow up with other projects, with more back-and-forth questions to discover what comments are "helpful" along the way.

File: fsharp-backend/src/ApiServer/Api/APIOps.fs [hunk 1]
```diff
@@ -43,6 +42,10 @@ let empty : Op.AddOpResult =
 
 let causesAnyChanges (ops : PT.Oplist) : bool = List.any Op.hasEffect ops
 
+/// API endpoint to add a set of Operations in a Canvas
```
Comment (pbiggar) [inline fsharp-backend/src/ApiServer/Api/APIOps.fs:45]: These are properly called Ops, not Operations. Might be worth defining in Op.fs that an Op is an Operation, but I wouldn't use the full word here when we only use an abbreviation throughout the code.

File: fsharp-backend/src/ApiServer/Ui.fs [hunk 2]
```diff
@@ -1,3 +1,7 @@
+/// Supports loading of static HTML pages
```
Comment (pbiggar) [inline fsharp-backend/src/ApiServer/Ui.fs:1]: They're not totally static, as we replace per-user things.

Comment (pbiggar) [inline fsharp-backend/src/ApiServer/Ui.fs:1]:
```suggestion
/// Supports loading of HTML pages
```

**Review by pbiggar — CHANGES_REQUESTED** [review]
This is great! Very much in support of this.

I've made a few notes where appropriate, ready to merge once they're resolved.

**Comment by pbiggar** [comment]
> In future similar PRs, expect many /// comments of /// WHATISTHIS. I think that'll be a good way for me to ask many questions all at once, and document the responses in-code.

Nice idea

**Pull request merged**